cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -178,6 +178,44 @@ func generateInitramfsMounts() (err error) {
 	return nil
 }
 
+func doBaseBindMount() (err error) {
```

Comment:
since these functions only do the mounts if the mount units don't exist, then this should be called

```suggestion
func maybeDoBaseBindMount() (err error) {
```